trl/trainer/ppo_trainer.py

```diff
@@ -875,6 +879,17 @@ def create_model_card(self, path: str, model_name: Optional[str] = "TRL Model")
         with open(os.path.join(path, "README.md"), "w", encoding="utf-8") as f:
             f.write(model_card_content)
 
+    def set_seed(self, seed: Optional[int]):
```

Comment:
I would move the function to core.py and expose it to the user via `__init__.py`. So if we wanted we can use it in the script too.